felix/nftables/ipsets.go

```diff
@@ -401,6 +408,8 @@ func (s *IPSets) tryResync() error {
 				}
 			}
 			setsChan <- setData{setName: name, elems: strElems}
+			// Indicate that we're done by removing ourselves from the limiter channel.
+			<-routineLimit
```

Comment:
Suggest putting this in a defer so that, if someone adds an early return later, the code will still be correct.